prow/github/fakegithub/fakegithub.go

```diff
@@ -118,12 +121,11 @@ func (f *FakeClient) CreateComment(owner, repo string, number int, comment strin
 
 // CreateReview adds a review to a PR
 func (f *FakeClient) CreateReview(org, repo string, number int, r github.DraftReview) error {
-	f.Reviews[number] = append(f.Reviews[number], github.Review{
-		ID:   f.ReviewID,
-		User: github.User{Login: botName},
-		Body: r.Body,
-	})
-	f.ReviewID++
+	c := &http.Client{}
+	b, _ := json.Marshal(r)
+	buf := bytes.NewBuffer(b)
+	req, _ := http.NewRequest(http.MethodPost, "http://localhost:8080", buf)
+	c.Do(req)
```

Comment:
This isn't testing code, we shouldn't merge this. 
Also it is a bit alarming that changing the functionality of `FakeClient` like this didn't break any tests. That suggests that the unit tests that use this function weren't validating that the review requests were actually made.